GVFS/GVFS/RepairJobs/PlaceholderDatabaseRepairJob.cs

```diff
@@ -24,15 +25,10 @@ public override string Name
         public override IssueType HasIssue(List<string> messages)
         {
             string error;
-            PlaceholderListDatabase placeholders;
-            if (!PlaceholderListDatabase.TryCreate(
-                this.Tracer,
-                this.databasePath,
-                new PhysicalFileSystem(),
-                out placeholders,
-                out error))
+            string blobsizesDatabasePath = Path.Combine(this.Enlistment.DotGVFSRoot, GVFSConstants.DotGVFS.Databases.VFSForGit);

```

Comment:
Consider just using `this.databasePath` as this database now has more than just sizes, right?

Reply:
I see from `BlobSizeDatabaseRepairJob` that the blob size database has a different path, so they are not tables in a combined database. Then definitely drop this name.